cmd/ovnk-mcp-server/main.go

```diff
@@ -34,10 +35,18 @@ func main() {
 		if err != nil {
 			log.Fatalf("Failed to create OVN-K MCP server: %v", err)
 		}
-		log.Println("Adding tools to OVN-K MCP server")
+		log.Println("Adding Kubernetes tools to OVN-K MCP server")
 		k8sMcpServer.AddTools(ovnkMcpServer)
 	}
 
+	// Always add sosreport tools (they work offline with sosreport path parameter)
+	sosreportServer, err := sosreportmcp.NewMCPServer()
+	if err != nil {
+		log.Fatalf("Failed to create sosreport MCP server: %v", err)
+	}
+	log.Println("Adding sosreport tools to OVN-K MCP server")
+	sosreportServer.AddTools(ovnkMcpServer)
+
```

Comment:
I think we should add the sosreport MCP server to the offline mode. When the MCP server is deployed in-cluster, we would still be adding all the sosreport tools, which ideally shouldn't happen

Reply:
I think this can be achieved by supporting another value (say `both`) for the `mode` flag. When this mode is chosen, both live-cluster and offline mode tools are added to the MCP server.